youtube_dl/extractor/twitch.py

```diff
@@ -206,7 +206,8 @@ class TwitchChapterIE(TwitchItemBaseIE):
 
 class TwitchVodIE(TwitchItemBaseIE):
     IE_NAME = 'twitch:vod'
-    _VALID_URL = r'%s/[^/]+/v/(?P<id>\d+)' % TwitchBaseIE._VALID_URL_BASE
+    _VALID_URL_BASE = r'https?://(?:www\.|player\.)?twitch\.tv'
```

Comment:
I wasn't sure if the _VALID_URL_BASE from the BaseIE class was ever used elsewhere, so I updated it in TwichVodIE just to keep things consistent. I can remove it if it's not necessary.